env.yaml

```diff
@@ -0,0 +1,18 @@
+MEMBRANE_CORS_ALLOWED_ORIGINS:
+  type: str
+  field_info:
+    default: http://localhost:3000
```

Comment:
Missing required vs optional, descriptive text and link to documentation listing valid values.

Reply:
Added an extra attribute for `optional`.